### PR TITLE
Base on plugwise v0.8.0

### PR DESCRIPTION
**Pull request by bouwew**

### CODE REVIEW

File: custom_components/plugwise/const.py [hunk 1]
```diff
@@ -152,7 +155,7 @@ def _energySensor(
     "outdoor_temperature": _temperatureSensor(),
     "setpoint": _temperatureSensor(),
     "temperature": _temperatureSensor(),
-    "temperature_difference": _temperatureSensor(),
+    "temperature_difference": _temperatureSensor(False),
```
Comment (CoMPaTech) [inline custom_components/plugwise/const.py:158]: Intentional?

Comment (bouwew) [reply custom_components/plugwise/const.py]: Yes, a pretty useless sensor so disabled.

**Review by CoMPaTech — APPROVED** [review]
I originally had const like this but I tend to dislike repeating things. Lets do it this way and see what upstreaming recommends later on

**Review by CoMPaTech — APPROVED** [review]

**Comment by brefra** [comment]
Added some additional const refacturing and an correction to one of the previous commits in [cabe50b](https://github.com/plugwise/plugwise-beta/pull/133/commits/cabe50bce590757e193a2f0d102d3574514deb16)

**Review by brefra — APPROVED** [review]

**Review by CoMPaTech — APPROVED** [review]

**Comment by CoMPaTech** [comment]
As requested tested against P1 with the proposed new function (now snakestyled naming), also corrected the kWh units for the cumulatives.

**Review by brefra — APPROVED** [review]

**Pull request merged**